Lib/ufoLib/glifLib.py

```diff
@@ -628,7 +628,10 @@ def writeGlyphToString(glyphName, glyphObject=None, drawPointsFunc=None, formatV
 		_writeLib(glyphObject, root, validate)
 	# return the text
 	tree = etree.ElementTree(root)
-	text = etree.tostring(root, encoding=unicode, pretty_print=True)
```

Comment:
by the way, this was also doubly wrong, because if anything it should have been `encoding="unicode"` (as a string), not the built-in `unicode` function.

Reply:
Not according to the lxml doc I read, but on vacation right now so can’t dig the reference up, if it works the other way, great

Reply:
@benkiel you’re right, you can also pass the unicode function to the encoding argument:

https://github.com/lxml/lxml/blob/0ef938692ea222d1b15939e0356739b895e883dd/src/lxml/etree.pyx#L3318

I was misled by the fact that in the official docs I linked earlier there is no mention of that, only of “unicode” as a string.

Nevertheless, we need to restore the xml declaration, otherwise attempting to order the xml with another parser different from xml might not work properly.